app/controllers/catalog_controller.rb

```diff
@@ -393,18 +393,8 @@ def start_new_search_session?
   end
 
   class << self
-    # Blacklight's highlighting feature assumes that the metadata exists in the page
-    # and replaces the rendered version from the dopcument with that of the highlighting
-    # section. In the case of our full text field, we do not render it in the normal results
-    # so we need to not display the field at all unless it was returned in the highlighting.
-    def full_text_highlight_exists_in_response?(context, _config, document)
-      response = context.instance_variable_get(:@response) || {}
-      document_highlight = response.dig('highlighting', document['id'])
-      return true if document_highlight.present? && document_highlight.any? do |field, values|
-        Settings.full_text_highlight.fields.include?(field) && values.present?
-      end
-
-      false
+    def document_has_full_text_and_search_is_query?(context, _config, document)
+      context.params[:q].present? && document['full_text_tesimv'].present?
```

Comment:
Does this mean we have to return the whole `full_text_tesimv` field for every request? Maybe we'll need to introduce a new field (or look into some solr-side magic) to avoid that.

Reply:
I was concerned/worried about the same thing, but it appears to work?

The search results in our app appear to know if the `full_text_tesimv` is present or not (which was surprising after looking at the solr configs).  I guess it's possible that our local app configs don't match what's on the server, but that's the behavior that I'm getting in development. 

My other thought was to add a boolean field into the document, but not having to modify the indexing seemed enticing.